redbot/cogs/mod/kickban.py

```diff
@@ -173,6 +173,17 @@ class KickBanMixin(MixinMeta):
             await ctx.send(_("I cannot do that due to discord hierarchy rules"))
             return
         audit_reason = get_audit_reason(author, reason)
+        if reason is None:
+            reason = _("No reason was given.")
+        toggle = await self.settings.guild(guild).toggle_dm()
+        if toggle:
+            if guild.me.top_role > user.top_role and user != guild.owner and author != user:
+                with contextlib.suppress(discord.HTTPException):
+                    em = discord.Embed(
+                        title=_(bold("**You have been kicked from {guild}.**")).format(guild=guild)
```

Comment:
```suggestion
                        title=_(bold("You have been kicked from {guild}.")).format(guild=guild)
```

Pretty sure the `bold()` function itself will add the asterixes here.